contracts/ts/genMaciState.ts

```diff
@@ -82,24 +82,27 @@ export const genMaciStateFromContract = async (
       });
     });
 
-    deployPollLogs.forEach((event) => {
+    // eslint-disable-next-line @typescript-eslint/prefer-for-of
+    for (let j = 0; j < deployPollLogs.length; j += 1) {
```

Comment:
Why did you change it here?

Reply:
because we are calling a async function to get the contract addresses, and async doesn't play well with forEach